integration-tests/models/__snapshots__/test_flash_mixtral/test_flash_mixtral_all_params.json

```diff
@@ -64,7 +64,7 @@
       },
       {
         "id": 369,
-        "logprob": -0.06585693,
+        "logprob": 0.0,
```

Comment:
Huh ?

Reply:
Guess that's what a temperature of 0.5 does though:

```
❯ git grep '"logprob": 0\.0' | uniq | head -n5
integration-tests/models/__snapshots__/test_bloom_560m/test_bloom_560m.json:        "logprob": 0.0,
integration-tests/models/__snapshots__/test_bloom_560m/test_bloom_560m_all_params.json:        "logprob": 0.0,
integration-tests/models/__snapshots__/test_bloom_560m_sharded/test_bloom_560m_sharded.json:        "logprob": 0.0,
integration-tests/models/__snapshots__/test_flash_awq/test_flash_llama_awq_all_params.json:        "logprob": 0.0,
integration-tests/models/__snapshots__/test_flash_deepseek_v2/test_flash_deepseek_v2_all_params.json:        "logprob": 0.0,
```